### ISSUE TITLE
Add Virtual Location

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#77]
Add fields for virtual location name and url.  The feature of "add more details" will be implemented later.

**Comment by saumier** [comment]
@troughc Please test in staging

**Comment by troughc** [comment]
as noted in a new issue I created and assigned to you - Place is missing from Location @saumier - should we merge these two issues into one? (issue footlight-admin #163)

**Comment by troughc** [comment]
@saumier also, I did test the virtual location in my test event (caitlin's test event) and this is what I got on the front end (Screenshots). Note that the virtual location doesn't appear on the list page, but does on the event page. Note that the PLACE still appears on the list page, but not on the event details page nor in the JSON-LD.

![Image](https://user-images.githubusercontent.com/64489312/212997526-eb574a6c-5b8b-4a97-91d7-3d12f2f8d202.png)



![Image](https://user-images.githubusercontent.com/64489312/212997556-ca731089-716e-4c92-8563-c1faee7fd227.png)



![Image](https://user-images.githubusercontent.com/64489312/212996868-ae45c572-ef12-4f4b-93fd-89565e9622de.png)
